ldcbench.ckan-node/src/main/java/org/dice_research/ldcbench/nodes/ckan/simple/SimpleCkanComponent.java

```diff
@@ -154,7 +154,7 @@ private void addCloudNode(int node) throws Exception {
         boolean success = false;
         while (!success) {
             try {
-                addDataSource(tripleCreator.createNode(0, -1, -2, false).toString());
+                addDataSource(tripleCreator.createNode(0, -1, -2, false, false).toString());
```

Comment:
You could have overloaded the method, i.e., one version is your newly implemented version of `createNode`. The other could have the "old" set of parameters and could simply assume that the blank node flag is false. This second method would then simply call your new method. This could have saved you the effort to change the calls everywhere. :thinking: